en/docs/learn/managing-sms-templates-from-the-registry.md

```diff
@@ -0,0 +1,98 @@
+# Managing SMS Notification Templates From the Registry
+
+WSO2 Identity server supports notifications via SMS from WSO2 Identity Server 5.10.0 onwards. This document will 
+guide you through the process of adding a new SMS notification template and editing an existing notification template. 
+
+!!! info Before you begin 
+    An SMS template can only have following attributes. 
+    
+        - displayName
+        - type
+        - locale
+        - body
+    
+    ![sample-sms-template](../assets/img/learn/sms-templates/sample-sms-template.png)    
+        
+    Having more attributes such as `Footer` or `Subject` will lead for errors.
+    
+## Adding an SMS notification template
+
+In WSO2 Identity server, the notification templates are packed by default. But if you want to add a new SMS 
+notification template, following methods are available.
+
+1. Adding via the configuration files.
+2. Adding via the registry of the management console.
+
+### Adding an SMS notification template using configuration files
+
+1. Navigate to <IS_HOME>/repository/conf/sms folder.
+
+2. Click on the sms-templates-admin-config.xml file to view default templates and edit them.
+
+3. Add a new template with allowed attributes. A sample template is given below.
+
+     ![sample-sms-template](../assets/img/learn/sms-templates/sample-sms-template.png)   
+
+!!! note
+    If the identity server is running, you need to restart the server to apply the changes.       
+     
+### Adding an SMS notification template using the registry
+
+1. Log in to the management console and navigate to `Main> Registry> Browse`.
```

Comment:
**Main > Registry > Browse** should be in bold and not `monospace`.

Anything that is clickable on the management console should be in Bold. Please apply to all instances